src/framework/shortcuts/internal/shortcutsregister.cpp

```diff
@@ -186,7 +190,7 @@ Shortcut ShortcutsRegister::readShortcut(framework::XmlReader& reader) const
         } else if (tag == STANDARD_KEY_TAG) {
             shortcut.standardKey = QKeySequence::StandardKey(reader.readInt());
         } else if (tag == SEQUENCE_TAG) {
-            shortcut.sequence = reader.readString();
+            shortcut.sequence += reader.readString() + ";";
```

Comment:
quite a lot of commands have multiple alternative sequences defined, including importantly "delete" for which you can use both "Del" and "Backspace". Existing code was only ever taking the last.